### DISCUSSION TITLE
Github Externship

### DISCUSSION THREAD

**Discussion by AdityaAggarwal11** [github/india#585] [Students]
How much time it takes Github Externship to review after applying for Github Externship?
And also, please tell that what all things we will be doing in this internship and is it  good for begineers?

**Comment by punitjain03** [comment]
Hi Aditya, Thank you for applying at Github India Externship. Your profile will under review for 48 Hours to 72 Hours. Meanwhile keep exploring the organisation and check their tech-stacks :)

**Reply by raymax0x** [reply]
Hey Punit, I am 2021 Cs graduate, Can i be part of github Externship ??

**Comment by Bhavyaseth-afk** [comment]
How can we see the languages which the companies are using to build their projects? Will it be after the review process? 

Thanks

**Reply by punitjain03** [reply]
Once your profile is approved, You will be able to see the entire details including tech stacks. Meanwhile keep visit their website and gitHub repos

**Comment by arpitBhalla** [comment]
My College isn't listed Can I select another college?

**Reply by Isha307** [reply]
No, you can't. Because they will ask for college verification also.

**Reply by punitjain03** [reply]
No, Best is to ask your college to be a part of Github Campus Program, So you won';t miss the summer cohort.

**Comment by mayank-cse** [comment]
Hi Everyone,
*I am a second year student...am I eligible
*I wanted to know will they be accepting all the application or will shortlist profiles

**Reply by punitjain03** [reply]
Github externship is only open for Pre Final Year and Final year students.

**Comment by Pawanupadhyay10** [comment]
Hey, my college applied for the GitHub campus program today , but the name is not on the list Can I apply , and what is the last date to apply for the GitHub externship ??

**Reply by punitjain03** [reply]
15 Dec is the last date to submit your details to the orgs,

**Comment by Pawanupadhyay10** [comment]
I'm in final year .

**Reply by punitjain03** [reply]
Apply then

**Comment by sourabh-bhatt** [comment]
How's the difficulty of projects?

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by Raffi-stang-seher** [comment]
I’m learning Arduino and also practicing my English. I’m looking for simple game project ideas. Maybe we can brainstorm together?
